website/src/controllers/api-v3/user.js

```diff
@@ -31,4 +37,39 @@ api.getUser = {
   },
 };
 
+/**
+ * @api {post} /user/update-email
+ * @apiVersion 3.0.0
+ * @apiName EmailUpdate
+ * @apiGroup User
+ *
+ * @apiSuccess {Object} { status: 'ok' }
+ **/
+api.updateEmail = {
+  method: 'POST',
+  middlewares: [authWithHeaders(), cron],
+  url: '/user/update-email',
+  async handler (req, res) {
+    let user = res.locals.user.toJSON();
```

Comment:
@piousbox you should not call toJSON here but just do `let user = res.locals.user;`. This way `user` will be a mongoose document that you can `.save()`, `.remove()`, ... `.toJSON()` instead convert the document to a plain javascript object ready to be used as a response and will remove some private fields like `hashed_password`.